spec.md

```diff
@@ -1192,6 +1345,56 @@ message NodeGetIdResponse {
   // CO in subsequent `ControllerPublishVolume`.
   // This is a REQUIRED field.
   string node_id = 1;
+
+
+  // The topological segment(s) the provisioned volume is accessible
+  // from. A topological segment is a sub-division of a cluster, like
+  // region, zone, rack, etc.
+  // Each key must consist of alphanumeric characters, '-', '_' or '.'.
+  // Each value MUST contain 1 or more valid strings.
+  // A value containing more than 1 string indicates the volume is
+  // accessible from multiple topological segments.
+  // COs may use this information along with the topology information
+  // returned by NodeGetId to ensure that a given volume is accessible
+  // from a given node when scheduling workloads.
+  // This field is OPTIONAL. If it is not specified, the CO MAY assume
+  // the volume is equally accessible from all nodes in the cluster and
+  // may schedule workloads referencing the volume on any available node
+  //
+  // Example 1:
+  // Given a volume that is accessible from a single zone, and
+  // accessibility_requirements =
+  //   {"region":{"us-west1"}, "zone":{"us-west1-b"}}
+  // Indicates the volume is accessible only from the "region"
+  // "us-west1" and the "zone" "us-west1-b".
+  //
+  // Example 2:
+  // Given a volume that is accessible from two zones, and
+  // accessibility_requirements =
+  //   {"region": {"us-west1"}, "zone": {"us-west1-b", "us-west1-c"}}
+  // Indicates the volume is accessible from both "zone" "us-west1-b"
+  // and "zone" "us-west1-c" the "region" "us-west1".
+
+  // The topological segment(s) this node belongs to. A topological
+  // segment is a sub-division of a cluster, like region, zone, rack,
+  // etc.
+  // Each key must consist of alphanumeric characters, '-', '_' or '.'.
+  // Each value MUST contain a valid string.
+  // COs may use this information along with the topology information
+  // returned in CreateVolumeResponse to ensure that a given volume is
+  // accessible from a given node when scheduling workloads.
+  // This field is OPTIONAL. If it is not specified, the CO MAY assume
+  // the node is not subject to any topological constraint and may
+  // schedule workloads referencing any volume without any topological 
```

Comment:
I think there's a couple of ways to read this sentence:

(1) the CO ... MAY schedule workloads that reference any volume V on this node, irrespective of any topological constraints declared for V.
(2) the CO ... MAY schedule workloads that reference any volume V, such that there are no topological constraints declared for V.

Which do you intend?

Reply:
2 was the intention. Thanks for the clarification. I will modify to your wording.